### PR TITLE
Update coreclr-debug nuget package references and dynamically generate project.json

### PR DESCRIPTION
**Pull request by chuckries**
dynamic generation of project.json is done in order to supply a the runtime ID of the platform we are on.

### CODE REVIEW

**Comment by chuckries** [comment]
#138

**Comment by chuckries** [comment]
@gregg-miskelly @caslan @DustinCampbell

File: src/coreclr-debug/main.ts [hunk 1]
```diff
@@ -237,6 +240,66 @@ function ensureAd7EngineExists(channel: vscode.OutputChannel, outputDirectory: s
     });
 }
 
+function writeProjectJson(): Promise<void> {
+    return new Promise<void>(function(resolve, reject) {
+        var projectJson = createProjectJson(CoreClrDebugUtil.getPlatformRuntimeId());
+        
+        fs.writeFile(path.join(_util.coreClrDebugDir(), 'project.json'), JSON.stringify(projectJson, null, 2), {encoding: 'utf8'}, function(err) {
+           if (err) {
+               return err.code;
```
Comment (chuckries) [inline src/coreclr-debug/main.ts]: needs to be reject(err.code)

**Comment by caslan** [comment]
LGTM, verified CentOS as well

**Pull request merged**